bin/accessibility-automation/cypress/index.js

```diff
@@ -0,0 +1,140 @@
+/* Event listeners + custom commands for Cypress */
+
+Cypress.on('test:before:run', () => {
+  try {
+    if (Cypress.env("IS_ACCESSIBILITY_EXTENSION_LOADED") !== "true") return
+    const extensionPath = Cypress.env("ACCESSIBILITY_EXTENSION_PATH")
+  
+    if (extensionPath !== undefined) {
+      new Promise((resolve, reject) => {
+        window.parent.addEventListener('A11Y_TAP_STARTED', () => {
+          resolve("A11Y_TAP_STARTED");
+        });
+        const e = new CustomEvent('A11Y_FORCE_START');
+        window.parent.dispatchEvent(e);
+      })
+    }
+  } catch {}
+  
+});
+
+Cypress.on('test:after:run', (attributes, runnable) => {
+  try {
+    if (Cypress.env("IS_ACCESSIBILITY_EXTENSION_LOADED") !== "true") return
+    const extensionPath = Cypress.env("ACCESSIBILITY_EXTENSION_PATH")
+    const isHeaded = Cypress.browser.isHeaded;
+    if (isHeaded && extensionPath !== undefined) {
+
+      let shouldScanTestForAccessibility = true;
+      if (Cypress.env("INCLUDE_TAGS_FOR_ACCESSIBILITY") || Cypress.env("EXCLUDE_TAGS_FOR_ACCESSIBILITY")) {
+
+        try {
+          let includeTagArray = [];
+          let excludeTagArray = [];
+          if (Cypress.env("INCLUDE_TAGS_FOR_ACCESSIBILITY")) {
+            includeTagArray = Cypress.env("INCLUDE_TAGS_FOR_ACCESSIBILITY").split(";")
+          }
+          if (Cypress.env("EXCLUDE_TAGS_FOR_ACCESSIBILITY")) {
+            excludeTagArray = Cypress.env("EXCLUDE_TAGS_FOR_ACCESSIBILITY").split(";")
+          }
+
+          const fullTestName = attributes.title;
+          const excluded = excludeTagArray.some((exclude) => fullTestName.includes(exclude));
+          const included = includeTagArray.length === 0 || includeTags.some((include) => fullTestName.includes(include));
+          shouldScanTestForAccessibility = !excluded && included;
+        } catch (error) {
+          console.log("Error while validating test case for accessibility before scanning. Error : ", error);
+        }
+      }
+      let os_data;
+      if (Cypress.env("OS")) {
+        os_data = Cypress.env("OS");
+      } else {
+        os_data = Cypress.platform === 'linux' ? 'mac' : "win"
+      }
+      let filePath = '';
+      if (attributes.invocationDetails.relativeFile !== undefined) {
```

Comment:
Can use the helper `utils.isUndefined` that has null check
https://github.com/browserstack/browserstack-cypress-cli/blob/45027801c24b09e55c2dfb46808b39c9a710e2c4/bin/helpers/utils.js#L594

Reply:
The file runs in the context of users test suite, didn't want to import another file/method for this file.